src/update_handler.c

```diff
@@ -773,6 +773,8 @@ static gboolean copy_adaptive_image_to_dev(RaucImage *image, RaucSlot *slot, GEr
 	g_return_val_if_fail(error == NULL || *error == NULL, FALSE);
 
 	if (g_strv_contains((const gchar * const*)image->adaptive, "block-hash-index")) {
+		g_info("Selected adaptive update mode 'block-hash-index");
```

Comment:
Perhaps

```suggestion
		g_info("Selected adaptive update method 'block-hash-index");
```

Reply:
Fun fact: I had this first but was then confused by ``No compatible adaptive mode found in '%s'`` that comes a few lines later.

We should make this consistent, I'll add another patch.